packages/core/review-workflows/admin/src/routes/settings/components/tests/WorkflowAttributes.test.tsx

```diff
@@ -61,7 +61,10 @@ describe('WorkflowAttributes', () => {
     await findByText('1 content type selected');
   });
 
-  it('should not render a collection-type group if there are no collection-types', async () => {
+  /**
```

Comment:
Same comment as before, is there anything we can do to fix this here rather than disabling? We're losing a lot of tests.

Reply:
The way the component is implemented, it's really hard to tell if the async process has finished, we'd be better off just relying on e2e tests for components like this that rely on APIs